recipes-app/node-red-node-random/node-red-node-random_0.4.1.bb

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Li Hua Qian <huaqian.li@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+inherit npm
+inherit node-red-module
+
+DESCRIPTION = "A set of random nodes for Node-RED"
```

Comment:
No that the new commit message is more telling than no message...

Reply:
This module is required to be pre-installed by one customer, actually we don't have too much knowledge about that, we only know they want to use it to generate random numbers in their flow, and don't want to install this node for every piece of their devices.

Reply:
Understood, but this here is an example image, not a final product image. The latter will need a layer on top anyway to perform the require customizations and hardenings.

Reply:
Anyway, I'm not against adding it, specifically as it comes from an already used source (https://github.com/node-red/node-red-nodes), just clarify the commit message.

Reply:
Commit message refined